.github/workflows/ci.yaml

```diff
@@ -10,27 +10,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['8.8']
+        ghc: ['8.8', '8.10']
         os: [ubuntu-latest, macOS-latest]
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-haskell@v1.1
+    - uses: actions/setup-haskell@v1
```

Comment:
@mbj I'm curious, why the downgrade?

Reply:
@snusnu its actually an upgrade, this tag is supposed to point at the latest `v1` release, see: https://github.com/actions/setup-haskell/tags

Reply:
@mbj Ah, sorry for the noise then. I should've been able to find that out myself.

Reply:
@snusnu In fact `setup-haskell` used to slip on updating the "sliding tag". They just recently stepped up their game.